components/payments/settings/payment-webhooks.tsx

```diff
@@ -0,0 +1,242 @@
+import {
+  usePaymentsWebhooksByAccountId,
+  usePaymentsCreateWebhook,
+  usePaymentsUpdateWebhook,
+  type CreateWebhookInput,
+  type UpdateWebhookInput,
+  isValidWebhookUrl,
+} from "@3rdweb-sdk/react/hooks/usePayments";
+import type { PaymentsWebhooksType } from "@3rdweb-sdk/react/hooks/usePayments";
+import { Flex, Divider, useColorModeValue } from "@chakra-ui/react";
+import {
+  Card,
+  Heading,
+  CodeBlock
+} from "tw-components";
+import { PaymentsWebhooksTable, PaymentsWebhooksTableProps } from "./payments-webhooks-table";
+import React from "react";
+import { DetailsRow } from "components/settings/ApiKeys/DetailsRow";
+import { randomBytes } from "ethers/lib/utils";
+import {
+  Alert,
+  AlertTitle,
+  AlertIcon,
+  Text,
+  AlertDescription
+} from "@chakra-ui/react";
+
+interface PaymentsWebhooksProps {
+  accountId: string;
+}
+
+
+export const PaymentsWebhooks: React.FC<PaymentsWebhooksProps> = ({
+  accountId,
+}) => {
+
+  const [productionWebhooks, setProductionWebhooks] = React.useState<PaymentsWebhooksType[]>([]);
+  const [testnetWebhooks, setTestnetWebhooks] = React.useState<PaymentsWebhooksType[]>([]);
+
+  const { data: webhooks, isLoading: isGetLoading, isFetched, error } = usePaymentsWebhooksByAccountId(accountId);
+  const { mutate: updateWebhook, isLoading: isUpdateLoading, error: updateError, isSuccess: isUpdateSuccess } = usePaymentsUpdateWebhook(accountId);
+  const { mutate: createWebhook, isLoading: isCreateLoading, error: createError, isSuccess: isCreateSuccess } = usePaymentsCreateWebhook(accountId);
+
+  const [isUpdateDelete, setIsUpdateDelete] = React.useState(false);
+  const [currentDeleteWebhook, setCurrentDeleteWebhook] = React.useState<PaymentsWebhooksType>();
+  const [currentUpdateWebhook, setCurrentUpdateWebhook] = React.useState<PaymentsWebhooksType>();
+
+  const [currentCreateWebhook, setCurrentCreateWebhook] = React.useState<PaymentsWebhooksType>();
+
+  const [showAlert, setShowAlert] = React.useState(false);
+  const [alertHeader, setAlertHeader] = React.useState("");
+  const [alertDescription, setAlertDescription] = React.useState("");
+  const [alertStatus, setAlertStatus] = React.useState<"error" | "success">();
+
+  const setAlertTimeout = () => {
+    setTimeout(() => {
+      setShowAlert(false);
+      setAlertHeader("");
+      setAlertDescription("");
+      setAlertStatus(undefined);
+    }, 3000);
+  }
+
+  const triggerAlert = (status: "error"|"success", header: string, description: string) => {
```

Comment:
also realized for what you're doing here you probably want to use a toast instead?